frameworks/MLPlan/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
+VERSION=${1:-"latest"}
+
+# creating local venv
+. $HERE/../shared/setup.sh $HERE
+
+if [[ -x "$(command -v apt-get)" ]]; then
+    SUDO apt-get install -y wget unzip openjdk-11-jdk
+fi
+
+pip3 install --no-cache-dir -r $HERE/requirements.txt
+
+MLPLAN_ARC="mlplan.zip"
+DOWNLOAD_DIR="$HERE/lib"
+TARGET_DIR="$DOWNLOAD_DIR/mlplan"
+
+if [[ ! -e "$TARGET_DIR" ]]; then
+    mkdir -p $DOWNLOAD_DIR
+    echo "Download ML-Plan from extern"
+    wget http://127.0.0.1/mlplan/version/$VERSION -O $DOWNLOAD_DIR/$MLPLAN_ARC
```

Comment:
`127.0.0.1` is not going to work for everyone ;)

Reply:
On the contrary, it works even without a network connection!

Reply:
premise: I have a local build
proposition: all local builds can be downloaded
conclusion: I can download the build

Reply:
Sorry, on my working laptop I have Windows as an OS and thus needed to transfer changes to a Linux machine. I will change the URL asap again.

Reply:
Don't worry, it was some lighthearted fun :) these things slip in occasionally